.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -0,0 +1,37 @@
+name: Standard Issue Tempalte
+description: Create an Issue
+title: "[Issue]:"
+labels: ["research"]
+assignees:
+  - kitsushadow
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill this out best you can
```

Comment:
I think this could be a bit clearer. Something like "Describe an issue, idea, task, bug, or story."